src/puppetlabs/trapperkeeper/config.clj

```diff
@@ -59,6 +59,12 @@
     #{".yaml" ".yml"}
     (yaml/parse-string (slurp file))))
 
+(defn override-restart-file-from-cli-data
+  [config-data cli-data]
+  (if-let [cli-restart-file (:restart-file cli-data)]
+    (assoc-in config-data [:global :restart-file] cli-restart-file)
```

Comment:
I added a commit which logs the warning, 2c0648d.  The one somewhat annoying thing about it is that this function ends up getting logged twice at startup because this function is called from two places at startup in the TK code:

https://github.com/puppetlabs/trapperkeeper/blob/1.5.0/src/puppetlabs/trapperkeeper/core.clj#L125

This one is logged before global logback configuration is done by Trapperkeeper.

https://github.com/puppetlabs/trapperkeeper/blob/1.5.0/src/puppetlabs/trapperkeeper/config.clj#L103

This one is logged post Trapperkeeper logback reconfiguration.

Seems like this would come up so seldom that this isn't a big deal but just wanted to point it out.